setup.py

```diff
@@ -332,7 +332,7 @@ def is_Numpy_installed():
     'Bio.Compass',
     'Bio.Crystal',
     'Bio.Data',
-    'Bio.EBI.dbfetch',
+    'Bio.EBI',
```

Comment:
Given you are using ``Bio/EBI/dbfetch/__init__.py`` then I think you need both ``Bio.EBI`` and ``Bio.EBI.dbfetch`` here in ``setup.py``

Alternatively and perhaps more simply, I think you could use ``Bio/EBI/dbfetch.py`` instead and just ``Bio.EBI``in ``setup.py``?